### DISCUSSION TITLE
Why my participant suddenly not work correctly?Is nvida drivers old?

### DISCUSSION THREAD

**Discussion by hengyue-dunhuang** [cyberbotics/wrestling#67] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/77217960/228133420-992a6ad0-d5ab-46df-b36f-940d08f43061.png)

**Comment by omichel** [comment, marked as answer]
We updated the simulation server machine, rebooted it, ran your controller again, and it [worked out without any error](https://github.com/cyberbotics/wrestling/actions/runs/4540641173/jobs/8001807284). So, I guess the problem was related to a system update.
